app/templates/static/contact-us.html

```diff
@@ -15,72 +15,79 @@
-                                    Newport
-                                    <br>
-                                    South Wales
-                                <br>
-                                    NP10 8XG
-                                </p>
-                            </div>
-                        </div>
-                        </section>
+                            <section class="section id">
+                                <h1 class="section__title saturn">
+                                    Contact us
+                                </h1>
+                                <div class="question">
+                                    <h2 class="question__title neptune">
+                                        Telephone
+                                    </h2>
+                                    <div class="question__description mars">
+                                        <p>{{helpers.telephone_number()}}</p>
+                                    </div>
+                                    <p>
+                                        {% if session %}
+                                        Please quote the reference <b>{{session.ru_ref}}</b> when contacting ONS about
+                                        this
+                                        survey.
+                                    </p>
+                                    <p>
+                                        Your Survey ID is <b>{{survey_id}}</b>.
+                                        {% endif %}
+                                    </p>
+                                    <div class="question__description mars">
+                                        <p>
+                                        <h3 class="question__title venus">Opening hours:</h3></p>
+                                        Monday to Thursday 8:30am – 5:00pm<br>
+                                        Friday 8:30am - 4:30pm<br>
+                                    </div>
+                                    <p>This office is closed during public holidays</p>
+                                    <h2 class="question__title neptune">
+                                        Email
+                                    </h2>
+                                    <div class="question__description mars">
+                                        <p>{{helpers.email_address(subject="{{ru_ref}}")}}</p>
+                                    </div>
+                                    <h2 class="question__title neptune">
+                                        Post
+                                    </h2>
+                                    <div class="question__description mars">
+                                        <p>
+                                            Office for National Statistics
+                                            <br>
+                                            Government Buildings
+                                            <br>
```

Comment:
Can you move the line breaks to the end of the line.
`Office for National Statistics<br>
Government Buildings<br>`